imxrt1060-hal/src/gpio.rs

```diff
@@ -212,6 +219,84 @@ where
     }
 }
 
+/// GPIO input interrupt configurations.
+///
+/// These configurations do not take effect until
+/// GPIO input interrupts are enabled.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[repr(u32)]
+pub enum InterruptConfiguration {
+    // TODO isn't it a bit repetitive to have "Sensitive" on all here?
```

Comment:
Style question here. Maybe we drop `Sensitive`?

Reply:
Yea it's repetitive. OK with me to drop it!

I copied it straight from the reference manual, but we should't need it to understand these values.